### PR TITLE
Add content\hi\docs\setup\production-environment\turnkey-solutions.md

### PR DESCRIPTION
**Pull request by ShivamTyagi12345**
in continuation of https://github.com/kubernetes/website/pull/29966
@anubha-v-ardhan

### CODE REVIEW

File: content/hi/docs/setup/production-environment/turnkey-solutions.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title:  टर्नकी क्लाउड सॉल्यूशंस
+content_type:  अवधारणा
```
Comment (divya-mohan0209) [inline content/hi/docs/setup/production-environment/turnkey-solutions.md:3]: I think अवधारणा is accreditation. While in the [English version](https://github.com/kubernetes/website/blob/dev-1.22/content/en/docs/setup/production-environment/turnkey-solutions.md), the content_type is listed as concept. Could you please confirm @anubha-v-ardhan if this content_type was approved anywhere?

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]

File: content/hi/docs/setup/production-environment/turnkey-solutions.md (hunk 1 above)
Comment (anubha-v-ardhan) [reply content/hi/docs/setup/production-environment/turnkey-solutions.md:3]: Nope the content_type has to be concept only. I have not approved this earlier anywhere.

File: content/hi/docs/setup/production-environment/turnkey-solutions.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+weight: 30
+---
+<!-- overview -->
+
+यह पृष्ठ कुबेरनेट्स प्रमाणित समाधान प्रदाताओं की एक सूची प्रदान करता है।हर एक प्रदाता पृष्ठ से आप सीख सकते हैं कि उत्पादन योग्य क्लस्टर को इंस्टॉल और सेटअप कैसे करें।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/setup/production-environment/turnkey-solutions.md:8]:
```suggestion
यह पृष्ठ कुबेरनेट्स प्रमाणित समाधान प्रदाताओं की एक सूची प्रदान करता है। हर एक प्रदाता पृष्ठ से आप सीख सकते हैं कि उत्पादन योग्य क्लस्टर को इंस्टॉल और सेटअप कैसे करें।
```

File: content/hi/docs/setup/production-environment/turnkey-solutions.md (hunk 1 above)
Comment (anubha-v-ardhan) [inline content/hi/docs/setup/production-environment/turnkey-solutions.md:3]:
```suggestion
content_type: concept
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

File: content/hi/docs/setup/production-environment/turnkey-solutions.md (hunk 1 above)
Comment (anubha-v-ardhan) [reply content/hi/docs/setup/production-environment/turnkey-solutions.md:3]: @ShivamTyagi12345 Please commit the change as content_type should be concept and not translated. Conversation will be automatically resolved then

**Comment by ShivamTyagi12345** [comment]
made the changes

File: content/hi/docs/setup/production-environment/turnkey-solutions.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: Turnkey Cloud Solutions
+content_type:  concept
+weight: 30
```
Comment (anubha-v-ardhan) [inline content/hi/docs/setup/production-environment/turnkey-solutions.md:4]:
```suggestion
title: Turnkey Cloud Solutions
content_type: concept
weight: 30
```

Comment (anubha-v-ardhan) [reply content/hi/docs/setup/production-environment/turnkey-solutions.md:4]: NIT: extra space

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Comment by ShivamTyagi12345** [comment]
resolved

**Comment by anubha-v-ardhan** [comment]
/lgtm
/approve

Thank you @ShivamTyagi12345 :)

**Pull request merged**